scripts/apps/vocabularies/controllers/VocabularyEditController.ts

```diff
@@ -10,16 +12,51 @@ VocabularyEditController.$inject = [
     'vocabularies',
     'metadata',
     'cvSchema',
+    'relationsService',
 ];
 
+interface IScope extends IScopeConfigController {
+    idRegex: string;
+    vocabulary: IVocabulary;
+    selectionTypes: IVocabularySelectionTypes;
+    closeVocabulary: () => void;
+    updateVocabulary: (result: any) => void;
+    issues: Array<any> | null;
+    _errorUniqueness: boolean;
+    errorMessage: string;
+    save: () => void;
+    requireAllowedTypesSelection: () => void;
+    addItem: () => void;
+    cancel: () => void;
+    model: any;
+    schema: any;
+    schemaFields: Array<any>;
+    itemsValidation: { valid: boolean };
+    removeItem: (index: number) => void;
+    customFieldTypes: Array<{id: string, label: string}>;
+    setCustomFieldConfig: (config: any) => void;
+    editForm: any;
+}
+
 const idRegex = '^[a-zA-Z0-9-_]+$';
 
-export function VocabularyEditController($scope, notify, api, vocabularies, metadata, cvSchema) {
+export function VocabularyEditController(
+    $scope: IScope, notify, api, vocabularies, metadata, cvSchema, relationsService,
+) {
     var origVocabulary = _.cloneDeep($scope.vocabulary);
 
     $scope.idRegex = idRegex;
     $scope.selectionTypes = VOCABULARY_SELECTION_TYPES;
 
+    if ($scope.matchFieldTypeToTab?.('related-content-fields', $scope.vocabulary.field_type)) {
+        // Insert default allowed workflows
+        if ($scope.vocabulary.field_options == null) {
+            $scope.vocabulary.field_options = {allowed_workflows: relationsService.getDefaultAllowedWorkflows()};
```

Comment:
When I open an existing field for editing, I see the options selected. They couldn't have been selected, because it's the first time I'm opening it for editing after checking out this PR and I haven't run upgrade scripts either.

Reply:
but for existing fields, it makes sense to have all options selected, as it's the current behavior they have (they allow any workflow state)

Reply:
Right, I thought they were limiting the options. In that case, it's probably fine, even though it's strange that boxes are ticked when there is nothing in the database.